sql/db-connection.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+function addElective($title, $description, $lecturer){
+	$conn = new PDO('mysql:host=localhost;dbname=db_name', 'root', '');
```

Comment:
Maybe you can extract host, dbname, username and password into separate variables. The code will be more readable and easy to maintain.